Manager/IndexManager.php

```diff
@@ -305,31 +299,95 @@ private function getUniqueIndexName()
     }
 
     /**
-     * Returns the live physical index name, verifying that it exists
+     * @param string $alias
+     *
+     * @return array
+     *
+     * @throws IndexOrAliasNotFoundException
+     */
+    private function getIndicesForAlias(?string $alias)
```

Comment:
As for the symfony code conventions, private methods should go after the public methods